MekHQ/src/mekhq/campaign/autoResolve/scenarioResolver/abstractCombatSystem/AcsSimpleScenarioResolver.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
It's 02:30 local, so I'm going to call it here. Leaving this message so I know where I was up to.